src/Marten/Services/SystemTextJsonSerializer.cs

```diff
@@ -55,8 +59,18 @@ public void Customize(Action<JsonSerializerOptions> configure)
             configure(_options);
             configure(_optionsDeserialize);
             configure(_withTypes);
+
+            SyncDocumentDeserializeOptions();
         }
 
+        private void SyncDocumentDeserializeOptions() =>
```

Comment:
camel casing on private/protected members to be a little more consistent

Reply:
Renamed :+1: